engines/bastion_katello/app/assets/javascripts/bastion_katello/activation-keys/details/activation-key-add-subscriptions.controller.js

```diff
@@ -26,7 +26,12 @@ angular.module('Bastion.activation-keys').controller('ActivationKeyAddSubscripti
             'organization_id': CurrentOrganization,
             'search': $location.search().search || "",
             'sort_order': 'ASC',
-            'available_for': 'activation_key'
+            'available_for': 'activation_key',
+            'resourceName': 'Subscriptions',
+            'emptyRowsMessage': "You currently don't have any Subscriptions to subscribe to," +
+                                " you can add Subscriptions by selecting 'Products' under" +
+                                " 'Content' in the main menu"
```

Comment:
Why move these out of the html and into this JS file?  To me these messages make more sense in the html.

Also, by doing it this way these messages are no longer translated.